### ISSUE TITLE
Adding weight information to Neptune

### ISSUE THREAD

**Issue by yang15** [bennybp/planets#12]
The weight is missing in Neptune. Suggesting adding such information.

**Comment by dgasmith** [comment]
Agreed, can you check in with the Earth file to make sure the format is consistent?

**Comment by yang15** [comment]
I could not find the right format as you suggested in Earth, instead I used format in Sun
